### PR TITLE
Release Notes for WebView2 118 Prerelease SDK

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article section for review:
https://review.learn.microsoft.com/microsoft-edge/webview2/release-notes?branch=pr-en-us-2783&tabs=dotnetcsharp#102065-prerelease

This PR also makes some minor formatting changes for consistency & maintainability.

AB#46014315

### CODE REVIEW

File: microsoft-edge/webview2/release-notes.md [hunk 1]
```diff
@@ -104,6 +104,50 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
 
+<!-- ====================================================================== -->
+## 1.0.2057-prerelease
+<!-- Prerelease number TBD -->
+
+Release Date: August 23, 2023
+
+[NuGet package for WebView2 SDK 1.0.2057-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.2057-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 118.0.2057.0 or higher.<!-- todo: is .0 correct?  usually a nonzero number -->
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:115]: @victorhuangwq @peiche-jessica - is ".0" correct?
WebView2 Runtime version 118.0.2057.0
Expect a number eg 
WebView2 Runtime version 118.0.2057.123
like all other sections in this webpage

File: microsoft-edge/webview2/release-notes.md [hunk 2]
```diff
@@ -104,6 +104,50 @@ For full API compatibility, this version of the WebView2 SDK requires Microsoft
...
+
+#### [WinRT/C#](#tab/winrtcsharp)
+
+* `CoreWebView2NewWindowRequestedEventArgs` Class
+    * [CoreWebView2NewWindowRequestedEventArgs.OriginalSourceFrameInfo Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2newwindowrequestedeventargs#originalsourceframeinfo?view=webview2-winrt-1.0.2057-prerelease&preserve-view=true)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ExperimentalNewWindowRequestedEventArgs2 interface](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalnewwindowrequestedeventargs2?view=webview2-1.0.2057-prerelease&preserve-view=true)
+    * [ICoreWebView2ExperimentalNewWindowRequestedEventArgs2::get_OriginalSourceFrameInfo](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalnewwindowrequestedeventargs2?view=webview2-1.0.2057-prerelease&preserve-view=true#get_originalsourceframeinfo)
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:140]:
```suggestion
    * [ICoreWebView2ExperimentalNewWindowRequestedEventArgs2::get_OriginalSourceFrameInfo property](/microsoft-edge/webview2/reference/win32/icorewebview2experimentalnewwindowrequestedeventargs2?view=webview2-1.0.2057-prerelease&preserve-view=true#get_originalsourceframeinfo)

```

File: microsoft-edge/webview2/release-notes.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md:115]: fixed:
image

File: microsoft-edge/webview2/release-notes.md (hunk 2 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: done

File: microsoft-edge/webview2/release-notes.md [hunk 3]
```diff
@@ -132,6 +132,56 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
 * Fixed a bug where the WebView2 content sometimes renders at the incorrect size after changing the display configuration (such as laptop sleeping; remoting; or connecting or disconnecting an external display).  (Runtime-only)  ([Issue 3429](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3429))
 * Fixed a bug where a bluescreen happens when using WebView2 apps on certain hardware configurations.  (Runtime-only)  ([Issue 3679](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3679))
 
+<!-- end of August 2023 Release SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2057-prerelease
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:139]: @victorhuangwq @peiche-jessica - what version # & release date?

File: microsoft-edge/webview2/release-notes.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 08/28/2023
+ms.date: 08/29/2023
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:9]:
```suggestion
ms.date: 08/30/2023
```
possibly

File: microsoft-edge/webview2/release-notes.md [hunk 5]
```diff
@@ -132,6 +132,57 @@ No additional APIs have been promoted to Stable and added in this Release SDK.
 * Fixed a bug where the WebView2 content sometimes renders at the incorrect size after changing the display configuration (such as laptop sleeping; remoting; or connecting or disconnecting an external display).  (Runtime-only)  ([Issue 3429](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3429))
 * Fixed a bug where a bluescreen happens when using WebView2 apps on certain hardware configurations.  (Runtime-only)  ([Issue 3679](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3679))
 
+<!-- end of August 2023 Release SDK -->
+
+
+<!-- ====================================================================== -->
+## 1.0.2065-prerelease
+
+Release Date: August 29, 2023
```
Comment (mikehoffms) [inline microsoft-edge/webview2/release-notes.md:141]:
```suggestion
Release Date: August 30, 2023
```
possibly

File: microsoft-edge/webview2/release-notes.md (hunk 3 above)
Comment (mikehoffms) [reply microsoft-edge/webview2/release-notes.md]: 1.0.2065-prerelease, Aug 30

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**